seadas-reader/src/main/java/gov/nasa/gsfc/seadas/dataio/L1AHawkeyeFileReader.java

```diff
@@ -108,7 +101,7 @@ public void addGeocoding(final Product product) throws ProductIOException {
                 lats = geoNcFile.findVariable("geolocation_data/latitude");
                 lons = geoNcFile.findVariable("geolocation_data/longitude");
 
-                //Use lat/lon with PixelGeoCoding
+                //Use lat/lon with TiePointGeoCoding
```

Comment:
I think this change in the comment is not correct. In the end still a PixelGeoCoding is created.

By the way, the GeoCodingFactory should not be used anymore. It is better to use the ComponentGeoCoding. Probably we will soon change the implementation of GeoCodingFactory or maybe remove it at all. The new one is faster and more accurate.